docs/dex-specification.md

```diff
@@ -693,6 +697,12 @@ The structure contain the following information about the pool:
 - the amount of PTN tokens created per block
 - the total amount of staked tokens
 
+###### User Limit
+
+The `PoolInfo` structure of the Staking contract specifies whether or not there is a limit set for users (`userLimit`). If there is a limit, the `poolLimitPerUser` contains the max amount of staked tokens per user. If no limit is set, that value is zero. The `numberBlocksForUserLimit` contains information on how many blocks are available for each user after the start block. 
```

Comment:
The `numberBlocksForUserLimit` contains information on how many blocks `poolLimitPerUser` is valid. For example, if numberBlocksForUserLimit is 86400 blocks, so after 86400 blocks users will not have any limits.